icubam/www/handlers/home.py

```diff
@@ -40,6 +53,16 @@ def initialize(self, config, db_factory):
     super().initialize(config, db_factory)
     self.regions = None
 
+  def get_disclaimer_url(self):
+    if self.config.server.has_key('disclaimer'):
+      path = self.config.server.disclaimer
+      # The user has already agreed, we skip.
```

Comment:
I don't understand this comment. This only checks paths existence not user agreement.

Reply:
Bad copy/paste comment. Fixed on 2769ea7